Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMinor(from: "1.5.0")),
         .package(url: "https://github.com/123FLO321/turf-swift.git", .upToNextMinor(from: "0.3.1")),
         .package(url: "https://github.com/123FLO321/S2Geometry.git", .upToNextMinor(from: "0.3.1")),
-        .package(url: "https://github.com/123FLO321/POGOProtos-Swift.git", .upToNextMinor(from: "1.19.0")),
+        .package(url: "https://github.com/123FLO321/POGOProtos-Swift.git", .upToNextMinor(from: "1.19.0"))
```

Comment:
This doesnt look right. First commit goes from .18.0 to .20.0 but after rebase its back to .19.0.